server/routes/objects.js

```diff
@@ -43,7 +48,7 @@ router.get('/:url', async (req, res) => {
 
     minioClient.bucketExists(bucketName, async err => {
       if (err) {
-        console.error(err);
+        log.error(err);
         res.send({
```

Comment:
The `emitMessage` function is used correctly here, but ensure that it handles cases where `sock` might be undefined or null to avoid potential errors.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*